src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.filework.FileReader;
+import core.basesyntax.filework.FileWriteServiceImpl;
+import core.basesyntax.filework.ReportCreatorService;
+import core.basesyntax.handler.BalanceTransaction;
+import core.basesyntax.handler.PurchaseTransaction;
+import core.basesyntax.handler.ReturnTransaction;
+import core.basesyntax.handler.SupplyTransaction;
+import core.basesyntax.handler.TransactionHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Transaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String DATA_FILE_PATH =
+            "src/main/java/core/basesyntax/resourses/DataFile.csv";
```

Comment:
resourses should be in main directory 
```suggestion
            "src/main/resourses/DataFile.csv";
```